make/compiler_flags

```diff
@@ -232,50 +232,55 @@ endif
 # should contain "tbb" or "tbb tbbmalloc tbbmalloc_proxy". Setting the
 # TBB_LIBRARIES variable overrides the default.
 
-TBB_BIN ?= $(MATH)lib/tbb
-TBB_RELATIVE_PATH ?= ../$(notdir $(TBB))
-TBB_BIN_ABSOLUTE_PATH = $(abspath $(TBB_BIN))
-TBB_ABSOLUTE_PATH = $(abspath $(TBB))
-
-ifeq ($(OS),Darwin)
-  TBB_LIBRARIES ?= tbb tbbmalloc tbbmalloc_proxy
-else
-  TBB_LIBRARIES ?= tbb
-endif
+ifndef SYS_LIBS
```

Comment:
I didn't realize this syntax part of make 3.81! Is that the version shipped out with RTools now? (We were hampered by Windows version, which was 3.79 for the longest time.)

Reply:
It looks like `ifdef` is used lower in the file, so I guess it's ok!